.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+#        go: [1.8, 1.9, '1.10']
+        go: [1.11, 1.12, 1.13, 1.14, 1.15, 1.16]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Go ${{ matrix.go }}
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ matrix.go }}
+#    - name: "Setup dependencies"
+#      if: ${{ matrix.go == '1.8' }}
```

Comment:
So the tests are currently not run against earlier go versions? No way to fix this?

Reply:
Fixed